crates/h264/src/sps.rs

```diff
@@ -340,35 +340,11 @@ pub struct Sps {
 }
 
 impl Sps {
-    /// Parsees an SPS from the input bytes.
+    /// Parses an Sps from the input bytes.
```

Comment:
We should split this file into a module. 

and then each sub type has its own file.